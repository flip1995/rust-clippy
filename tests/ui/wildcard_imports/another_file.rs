// run-rustfix
#![warn(clippy::wildcard_imports)]
#![allow(unused, clippy::unnecessary_wraps, clippy::let_unit_value)]

// Test for #10580, the lint should **not** ignore it.

fn foofoo() {}

mod outer {
    mod inner {
        use super::super::*;
        fn barbar() {
            let _ = foofoo();
        }
    }
}

fn main() {}
