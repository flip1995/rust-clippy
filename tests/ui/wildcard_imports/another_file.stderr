error: usage of wildcard import
  --> $DIR/another_file.rs:11:13
   |
LL |         use super::super::*;
   |             ^^^^^^^^^^^^^^^ help: try: `super::super::foofoo`
   |
   = note: `-D clippy::wildcard-imports` implied by `-D warnings`

error: aborting due to previous error

